solution/ui/regulations/eregs-vite/src/utilities/api.js

```diff
@@ -359,66 +406,71 @@ const getRegSearchResults = async ({
     page = 1,
     page_size = 100,
 }) => {
-    const response = await httpApiGet(`search?q=${encodeURIComponent(q)}&paginate=${paginate}&page_size=${page_size}&page=${page}`);
+    const response = await httpApiGet(
+        `search?q=${encodeURIComponent(
+            q
+        )}&paginate=${paginate}&page_size=${page_size}&page=${page}`
+    );
 
     return response;
 };
 
 // todo: make these JS style camel case
-const getSupplementalContent = async (
-    {
-        partDict,
-        categories,
-        q = "",
-        start,
-        max_results = 1000,
-        paginate = true,
-        page = 1,
-        cat_details = true,
-        page_size = 100,
-        location_details = true,
-        sortMethod = "newest",
-        fr_grouping = true,
-    }
-) => {
+const getSupplementalContent = async ({
+    partDict,
+    title,
+    categories,
+    q = "",
+    start,
+    max_results = 1000,
+    paginate = true,
```

Comment:
should we switch these to camelcasing here.  ESLINT does not like it, i know its outside the scope but we are touching this so it would make sense to clean it up.  if you feel it is outside of scope though I'm fine with that too.

Reply:
I think this is a good place where the snake_cased params could be switch to camelCase.  I think it's better as a tech debt story/PR though.  This block of params only changed because Prettier formatted it.